modules/sno-du-configuring-logging-locally-and-forwarding.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-configuring-logging-locally-and-forwarding_{context}"]
+= Configuring logging locally and forwarding
+
+To be able to debug a single node distributed unit (DU), logs need to be stored for further
+analysis.
+
+.Procedure
+
+* Use the following example to configure logging:
++
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+  name: instance
+  namespace: openshift-logging
+spec:
+  collection:
+    logs:
+      fluentd: {}
+      type: fluentd
+  curation:
+    type: "curator"
+    curator:
+      schedule: "30 3 * * *"
+    managementState: Managed
+---
```

Comment:
Maybe my confusion here. After installing Cluster Logging, the `ClusterLogging` CR named `instance` exists. The `ClusterLogForwarder` CR does not (unless this changed. Will this step work?

Reply:
@yliu127 @imiller0 Please confirm.

Reply:
These two CRs should be applied to the cluster to configure the logging. The ClusterLogging CR should be used to update the existing instance (or create it if it does not). Similarly the ClusterLogForwarder CR should be created if it does not exist, or updated if it does.

Reply:
I'm going to add callouts based on Ian's answer: